### PR TITLE
feat: allow to show multiple slides per view in the carousel

### PR DESCRIPTION
**Pull request by talamcol**
References [link the ticket here]

## Motivation and context

Adjustments needed to support multiple slides on a single carousel page for the new AI recommendations

## How to test

https://talamcol-ai-carousel-listings-web.branch.autoscout24.dev/de/ld/listings/8680706/message-lead/success

### CODE REVIEW

**Comment by talamcol** [comment]
QA approved by Migmar ✅

**Comment by talamcol** [comment]
verified in seller-web and listings-web

**Review by damirzecevicsmg — APPROVED** [review]
Looks good to me.

File: src/components/carousel/index.tsx [hunk 1]
```diff
@@ -16,6 +20,9 @@ type SharedProps = {
   startIndex?: number;
   onSlideClick?: (index: number) => void;
   onSlideSelect?: (index: number) => void;
+  slidesPerView?: ResponsiveObject<number>;
+  loop?: boolean;
+  slidesToScroll?: 'auto' | number;
```
Comment (damirzecevicsmg) [inline src/components/carousel/index.tsx:25]: It seems that slidesToScroll: auto and slidesToScroll: 1 work the same. Do we have a specific use case where these two works differently?

Comment (talamcol) [reply src/components/carousel/index.tsx:25]: no no they don't work the same. If you have multiple slides per view, it adjusts based on it.

**Pull request merged**